backend/app/main.py

```diff
@@ -160,10 +204,17 @@ async def get_document(document_id: str, session: Session = Depends(get_session)
             Document.created_at,
             Document.gerrydb_table,
             Document.updated_at,
-            GerryDBTable.tiles_s3_path.label("tiles_s3_path"),
-        )
+            DistrictrMap.parent_layer.label("parent_layer"),  # pyright: ignore
+            DistrictrMap.child_layer.label("child_layer"),  # pyright: ignore
+            DistrictrMap.tiles_s3_path.label("tiles_s3_path"),  # pyright: ignore
+            DistrictrMap.num_districts.label("num_districts"),  # pyright: ignore
+        )  # pyright: ignore
         .where(Document.document_id == document_id)
-        .join(GerryDBTable, Document.gerrydb_table == GerryDBTable.name, isouter=True)
+        .join(
+            DistrictrMap,
+            Document.gerrydb_table == DistrictrMap.gerrydb_table_name,
+            isouter=True,
+        )
         .limit(1)
```

Comment:
(pp) not caused by this PR, but just noticing- limit(1) is going to prevent run time errors if we do for some reason violate uniqueness constraints, but, uh, we have uniqueness constraints on the database. I think we should be consistent-- pair result.one() with a .limit(1) each time.

Reply:
Totally but I'd prefer to actually remove the limit and handle the runtime error, allowing us to send an informative error and catch bugs earlier. Bugs ofc not caused by erroneous db state-because we have our constraints in place-but in our queries